tests/Feature/Resources/Views/Pages/Partials/Header/PreviewTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Can you please put the tests into the given file `PageHomeTest`. I would like to keep them there.